### PR TITLE
Update some requirements

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

**Comment by djgalloway** [comment]
Bootstrap completed for me

**Comment by djgalloway** [comment]
`ERROR: No matching distribution found for boto3==1.21.20`

**Comment by zmc** [comment]
I'm seeing some very weird interactions between `pip` and `pip-compile`. The latter had picked a version of pytest that the former claimed didn't exist

**Comment by zmc** [comment]
Removed the pytest update; will address that separately

**Review by dmick — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --output-file=- requirements.in
 #
-ansible==2.9.24
+ansible==2.9.27
```
Comment (dmick) [inline requirements.txt:7]: dependabot beat you to this, and did an automated push?   :scream:

**Pull request merged**